docs/pypi.org/Terms-of-Use.md

```diff
@@ -1,7 +1,10 @@
 ---
 source: https://pypi.org/policy/terms-of-use/
+title: Terms of Use
 ---
-# Terms of Use
+# PyPI Terms of Use
+
+Superceded by [PyPI Terms of Service](Terms-of-Service.md): **TBD**
```

Comment:
```suggestion
Superseded by [PyPI Terms of Service](Terms-of-Service.md): **TBD**
```

What's the TBD going to be replaced with?

Reply:
TBD! Once we have the PR of warehouse ready we'll choose a date to roll all of this out.